CODEOWNERS

```diff
@@ -13,3 +13,4 @@
 #
 
 * @BogdanDrutu @pjanotti @sjkaris @rghetia @tedsuo
+
```

Comment:
I can't comment above (line 3), but I think it should read `List of approvers for OpenTelemetry golang SDK` instead of  `List of approvers for OpenTelemetry .NET SDK`